server_setup.coffee

```diff
@@ -290,8 +290,10 @@ setupQuickBailToMainHTML = (app) ->
 
       if req.headers.host is 'cp.codecombat.com'
         features.codePlay = true # for one-off changes. If they're shared across different scenarios, refactor
-      if /cn\.codecombat\.com/.test(req.get('host'))
+      if /cn\.codecombat\.com/.test(req.get('host')) or /koudashijie\.com/.test(req.get('host'))
```

Comment:
This is hard to test because it is using a `host`. It would be easier to test if this used a config flag.

If you've tested this and are confident then it's probably fine.